gen_errors/cbl-errors.json

```diff
@@ -34,6 +34,7 @@
     { "name": "DuplicateSelectResultName", "message": "Duplicate select result named {0}." },
     { "name": "NoAliasInJoin", "message": "The default database must have an alias in order to use a JOIN statement (Make sure your data source uses the As() function)." },
     { "name": "InvalidQueryDBNull", "message": "Invalid query: The database is null." },
-    { "name": "InvalidQueryMissingSelectOrFrom", "message": "Invalid query: missing Select or From." }
+    { "name": "InvalidQueryMissingSelectOrFrom", "message": "Invalid query: missing Select or From." },
+    { "name": "PullOnlyReplicatorsNotSupportPendingDocIds", "message": "Pending Document IDs are not supported on pull-only replicators." }
```

Comment:
suggestion on key, `PullOnlyReplicatorsNotSupportPendingDocIds`, seems complicated and difficult to grasp the meaning. 

how about `UnsupportedReplicatorForPendingDoc`

Reply:
Hmm, that sounds strange... it sounds like replicator is not supported for pending doc...

Reply:
@borrrden suggests `PullOnlyPendingDocIDs`. I think it's quite fine.